src/components/__stories__/login_header.tsx

```diff
@@ -5,5 +5,5 @@ import LoginHeader from "../login_header"
 
 storiesOf("LoginHeader", LoginHeader)
   .add("Simple login header", () => (
-    <LoginHeader/>
+    <LoginHeader title={"Welcome back, please log in to your account"}/>
```

Comment:
…that is, [this line](https://github.com/artsy/reaction-force/pull/32/files#diff-bf1946d7c4ad3781c0372789ab3432d0R18) and [this line](https://github.com/artsy/reaction-force/pull/32/files#diff-bf1946d7c4ad3781c0372789ab3432d0R20) are not needed when you allow the user of `LoginHeader` to just pass in their own title(s).